CreateMessage/handler.ts

```diff
@@ -307,6 +361,63 @@ export function CreateMessageHandler(
     const { service, email: serviceUserEmail } = userAttributes;
     const { authorizedRecipients, serviceId } = service;
 
+    // Check configuration id if third_party_data has been sent
+    const isRemoteContentMessage = !!messagePayload.content.third_party_data;
+    const hasRemoteContentMessageConfigurationId = !!messagePayload.content
```

Comment:
it would be better to parse the `messagePayload` to an appropriate "message variant" (`RemotedContentMessage`, with a dedicated codec) to centralize this constraint

Reply:
this would be particularly difficult and out of scope in this moment. Since we will have to migrate to a v2 version in the next future, I'd keep this kind of optimization in that job.